eth-node/types/waku.go

```diff
@@ -3,30 +3,43 @@ package types
 import (
 	"context"
 	"crypto/ecdsa"
+	"encoding/json"
 	"sync"
 	"time"
 
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/libp2p/go-libp2p/core/protocol"
+	"github.com/multiformats/go-multiaddr"
 	"github.com/pborman/uuid"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/p2p/enode"
 	"github.com/status-im/status-go/connection"
 )
 
 type ConnStatus struct {
-	IsOnline bool                  `json:"isOnline"`
-	Peers    map[string]WakuV2Peer `json:"peers"`
+	IsOnline bool      `json:"isOnline"`
+	Peers    PeerStats `json:"peers"`
+}
+
+type PeerStats map[peer.ID]WakuV2Peer
```

Comment:
This type was created so it's possible to marshal a map whose key is a `peer.ID`. Since this type is a `string`, `json.Marshal` is printing the directly the value of the `peer.ID` without doing a b58 encoding like it does when calling `peer.ID.String()`.